components/infobox/wikis/overwatch/infobox_person_player_custom.lua

```diff
@@ -177,4 +179,15 @@ function CustomPlayer:_isPlayerOrStaff()
 	end
 end
 
+---@return string?
+function CustomPlayer:createBottomContent()
+	if self:shouldStoreData(self.args) and String.isNotEmpty(self.args.team) then
+		local teamPage = Team.page(mw.getCurrentFrame(), self.args.team)
+		local team2Page = Team.page(mw.getCurrentFrame(), self.args.team2)
+		return
+			tostring(MatchTicker.player{recentLimit = 3}) ..
+			Template.safeExpand(mw.getCurrentFrame(), 'Upcoming and ongoing tournaments of', {team = teamPage}, {team2 = team2Page})
```

Comment:
```suggestion
			Template.safeExpand(
				mw.getCurrentFrame(),
				'Upcoming and ongoing tournaments of',
				{team = teamPage}, {team2 = team2Page}
			)
```
as an example how to break the long line into shorter lines to please the linter